docs/testnet_deployment.json

```diff
@@ -1,44 +1,55 @@
-              "bnUSD": "0xc0c1aA22F99bb6724dC4159C256A5989D90A659C",
-              "BMCManagement": "0x5B9733113745F87A790321e2a84e73CbDa628B95",
-              "BMCPeriphery": "0x853CceE29C20331DB18937E6bEdb8ab477ebe691",
-              "BTSCore": "0x1a2aDf985D6c2700fdAf72A9c1e2b39e3B647F7e",
-              "BTSPeriphery": "0x69555E113279b3d3805492a829DD765ED394A669"
-         }
-    }
+     "networks": {
+          "icon": {
+               "uri": "https://lisbon.net.solidwallet.io/api/v3/icon_dex",
+               "network_id": "0x2",
+               "btp_network_id": "0x2.icon",
+               "block_height": 11273953
+          },
+          "bsc": {
+               "uri": "https://data-seed-prebsc-1-s1.binance.org:8545",
+               "network_id": "0x61",
+               "btp_network_id": "0x61.bsc",
+               "block_height": 21985565
+          },
+          "near": {
+               "uri": "https://rpc.testnet.near.org",
+               "network_id": "0x2",
+               "btp_network_id": "0x2.near",
+               "block_height": 102880710
+          }
+     },
+     "contracts": {
+          "icon": {
+               "sICX": "cxc7b869d97a567044a2a5acea180ecd9071cd20b0",
+               "bnUSD": "cxcadcaf77d8e46089fd3d98fcf71eabee1700f148",
+               "BNB": "cx55b835590d43af7bf6f5be3c3d50982264d24e5d",
+               "BUSD": "cx39a7fefbe48ca7984e036abb1fab16f25fa641bb",
+               "USDT": "cxdb7e556a7fd5441bead5d7c5403d414223d51132",
+               "USDC": "cxf021731307d9f382935ac4bda81ee03946e1e4bd",
+               "BTCB": "cx7c1ef7b631a64e33e35565b81e669a05cdd0d968",
+               "ETH": "cx4663c5f1d955207c5718aedb11ff029f08b50036",
+               "bmc": "cx053b96e2de3b9e6fc06db7b390d96ebf5fe82892",
+               "bts": "cxa843db0a27750230559f997bafaeb7f8739afc81"
+          },
+          "bsc": {
+               "BUSD": "0x119344c4354AcD401Ffb7Ed58e2B03fBA6759a87",
+               "USDT": "0x51C59A4453e26c1DC46E282cbC6FFDD72ceC16DA",
+               "USDC": "0x2f84D2Eeac14e2d02aaf00eB25801721D4233d19",
+               "BTCB": "0xd5d707da717cf4771Ed995f16E976C711890F118",
+               "ETH": "0xE3471f58314332502Be2B0681a66D674D98763D6",
+               "ICX": "0x7d8c52A23FD7e3ca1342797baE7caF6d7b8036BA",
+               "sICX": "0x0a7792fe75548b26b287871081Aa6b05f48D9e89",
+               "bnUSD": "0xc0c1aA22F99bb6724dC4159C256A5989D90A659C",
+               "BMCManagement": "0x5B9733113745F87A790321e2a84e73CbDa628B95",
+               "BMCPeriphery": "0x853CceE29C20331DB18937E6bEdb8ab477ebe691",
+               "BTSCore": "0x1a2aDf985D6c2700fdAf72A9c1e2b39e3B647F7e",
+               "BTSPeriphery": "0x69555E113279b3d3805492a829DD765ED394A669"
+          },
+          "near": {
+               "bmc":"7270a79be789d770f2de015047684e2806597eeee96ee3ca87b179c6399deaaf",
+               "bts":"bts.iconbridge-6.testnet",
+               "ICX":"btp-icx.bts.iconbridge-6.testnet"
```

Comment:
please move tokens above smart contract addresses to be consistent with icon and bsc, and add an equivalent contract for each of the tokens listed on icon and bsc that are not currently listed on near

Reply:
Also, where are the NEAR token addresses for the other networks?